Source/Fuse.Controls.Panels/Backgrounds/TopFrameBackground.uno

```diff
@@ -49,7 +49,13 @@ namespace Fuse.Controls
 
 		protected override float2 GetContentSize(LayoutParams lp)
 		{
-			if defined(iOS || Android)
+			if defined(iOS) {
+				// on iOS, we always treat the bar as though it's 40 points
+				var height = 40 / AppBase.Current.PixelsPerPoint;
```

Comment:
This is also the issue that got flagged last time - the tricky issue is that it's device specific. We can calculate it by getting the height of the top frame, but there is no consistent measurement we can get at runtime that way which will mean this functions as expected.